translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -9468,32 +9468,27 @@ Value *SPIRVToLLVM::transTrinaryMinMaxExtInst(SPIRVExtInst *ExtInst,
   switch (ExtInst->getExtOp()) {
 
   case FMin3AMD: {
-    // Minimum of three FP values. Undefined result if any NaNs.
-    FastMathFlags FMF;
+    // Middle of three FP values. Undefined result if any NaNs.
```

Comment:
"Minimum"

Reply:
Oops. Too late as I have merged it, but I have pushed #239.